### PR TITLE
Remove store_stress_old and store_stress_older

### LINKED ISSUE #12574: Get rid of `store_stress_old` parameter in `ComputeStressBase`
## Rationale

It looks like the `store_stress_old` parameter in `ComputeStressBase.C` isn't used anywhere. I'm guessing it was used for something that was removed in the past, and this simply got missed in the cleanup.

## Description

I just stumbled across the `store_stress_old` parameter in `ComputeStressBase.C`, and was puzzled about why it existed (because the material models that need the old stress already call `getMaterialPropertyOld()` to get the old stress. It looks like that parameter and the member variable that it is stored in are not actually used anywhere.

## Impact

Should be a minor change, although there may be input files that set that parameter (even though it apparently doesn't do anything).

### PR DESCRIPTION
**Pull request by bwspenc**
closes #12574

### CODE REVIEW

**Comment by bwspenc** [comment]
@sveerara I haven't looked at Mastodon, but I'm guessing that this will fail the Mastodon tests. If you could remove all usage of 'store_stress_old' and 'store_stress_older' in Mastodon, that would be great.

File: modules/combined/test/tests/solid_mechanics/Rayleigh_damping/Newmark_time_integration/Rayleigh_Newmark.i [hunk 1]
```diff
@@ -20,9 +20,6 @@
 # Pressure is evaluated using Pressure boundary condition
 #
 # The system will come to steady state slowly after the pressure
```
Comment (sveerara) [inline modules/combined/test/tests/solid_mechanics/Rayleigh_damping/Newmark_time_integration/Rayleigh_Newmark.i]: becomes constant.

File: modules/combined/test/tests/solid_mechanics/Time_integration/HHT_time_integration/HHT_test.i [hunk 2]
```diff
@@ -20,9 +20,6 @@
 #
 # The system will come to steady state slowly after the pressure
 # becomes constant.  Alpha equal to zero will result in Newmark
```
Comment (sveerara) [inline modules/combined/test/tests/solid_mechanics/Time_integration/HHT_time_integration/HHT_test.i]: time integration.

**Review by sveerara — CHANGES_REQUESTED** [review]
I have made minor comments on the PR. The Dynamics documentation (Dynamics.md in modules/tensor_mechanics) also contains a couple of lines on using store_stress_old. Can you please remove that as well? Thanks!

**Comment by bwspenc** [comment]
@sveerara Sure, I'll fix those things.  Let me know when you get the Mastodon test fixes merged in, and then I'll push in my changes. That will trigger the tests getting run again, and they should pass at that point.

**Comment by sveerara** [comment]
@bwspenc, the Mastodon test fixes have been merged.

**Comment by bwspenc** [comment]
Thanks for fixing the Mastodon tests, @sveerara.  I just pushed an update that address all of your comments. Hopefully the tests will pass.

**Review by sveerara — APPROVED** [review]
Thanks for making these changes.

**Review by dschwen — APPROVED** [review]

**Pull request merged**